app/[envId]/products/page.tsx

```diff
@@ -47,9 +46,6 @@ const ProductsPage = async ({params}: {params: Promise<{envId: string}>}) => {
   )
 };
 
-
-export const generateStaticParams = () => [ { envId: defaultEnvId } ]
```

Comment:
How come that the `envId` param is not needed?

Reply:
because we are generating it in layout.tsx